src/main/java/seedu/address/model/commons/Name.java

```diff
@@ -16,7 +16,7 @@
      * The first character of the Name must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
-    public static final String VALIDATION_REGEX = "(\\p{Graph}+\\p{Space}?)+";
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
```

Comment:
What is the reason for this change? Previously I changed it to Graph so that it allows for names with some punctuations, such as "Alice's Home", i also tested it so it was working.

Reply:
This test will fail so i changed it back.
![image](https://user-images.githubusercontent.com/65444547/95451455-62ac3a80-099a-11eb-8f90-a1b323a89407.png)

Reply:
I see, then can change to `"(\\p{Alnum}p{Graph}*\\p{Space}?)+"` ?